src/CryptoAxus.Domain/Collections/OffersDocument .cs

```diff
@@ -38,6 +38,7 @@ public OffersDocument() : base()
                           ObjectId createdBy,
                           DateTime? lastModifiedDate = null,
                           ObjectId? lastModifiedBy = null,
+                          string? signature = null,
```

Comment:
This field was removed in the previous ticket.

Reply:
Resolved